lti_consumer/lti_consumer.py

```diff
@@ -425,6 +393,21 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         default=False,
         scope=Scope.settings
     )
+    user_username = String(
+        help=_("User's username; only used if ask_to_send_username is True, and user authorizes it."),
+        default=None,
+        scope=Scope.user_state
+    )
+    user_email = String(
+        help=_("User's email; only used if ask_to_send_email is True, and user authorizes it."),
+        default=None,
+        scope=Scope.user_state
+    )
```

Comment:
I think this has PII concerns. I'm not sure you should be storing this in the XBlock.

Reply:
Resolved in most recent commit